runtime/moonriver/src/lib.rs

```diff
@@ -674,7 +674,7 @@ impl pallet_parachain_staking::Config for Runtime {
 	/// Minimum stake required to become a collator
 	type MinCollatorStk = ConstU128<{ 1000 * currency::MOVR * currency::SUPPLY_FACTOR }>;
```

Comment:
Wasn't this discussed to be changed too ?

Reply:
Yeah, I realised that I should change this constant too. That´s what I did a few minutes ago and know I'm finishing fixing some tests.

Reply:
Shouldn't this imly that `MinCandidateStk >= MinCollatorStk`? That wasn't the case before this PR, but it seems to be implied:

```rust
        /// Minimum stake required for any candidate to be in `SelectedCandidates` for the round
        #[pallet::constant]
        type MinCollatorStk: Get<BalanceOf<Self>>;
        /// Minimum stake required for any account to be a collator candidate
        #[pallet::constant]
        type MinCandidateStk: Get<BalanceOf<Self>>;
```

CC @4meta5

Reply:
We should also be careful bumping `MinCollatorStk` because it could prevent a lot (theoretically all) candidates from being selected for future rounds

Reply:
I agree. For now I put the same value in both constants (your bond must be 10000 MOVR to both be selected for the round and be a collator candidate). We should discuss this in more detail and then I change the values and fix tests again if necessary.